app/Http/Resources/CicloResource.php

```diff
@@ -14,6 +14,8 @@ class CicloResource extends JsonResource
      */
     public function toArray(Request $request): array
     {
-        return parent::toArray($request);
+        return array_merge(
+            parent::toArray($request),
+            ['familia_id' => $this->familia_id]);
```

Comment:
En lugar de `'familia_id' => $this->familia_id`, deberías haber utilizado `'familia_id' => $this->familiaProfesional`

También te ha faltado modificar el `FamiliaProfesionalResource`